### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by fishinthecalculator**

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -1 +1,19 @@
...
+    import_config "../../bonfire/#{config_file}"
+
+  File.exists?("deps/bonfire/#{config_file}") ->
+    IO.puts("Load #{config_file} from `bonfire` dep")
+    import_config "../deps/bonfire/#{config_file}"
+
+  true ->
+    IO.puts("No #{config_file} found")
+    nil
+end
```
Comment (mayel) [inline config/config.exs]: bonfire isn't a dependency here right? So probably this section isn't needed?

**Review by mayel — CHANGES_REQUESTED** [review]

File: config/config.exs (hunk 1 above)
Comment (fishinthecalculator) [reply config/config.exs]: Good catch, thank you :) I should have fixed that

**Review by mayel — APPROVED** [review]

**Comment by mayel** [comment]
@fishinthecalculator I've made a change in bonfire_common so that localisation_path is in :bonfire_common instead of :bonfire, so now the config looks like this:
```elixir
config :bonfire_common, # use :bonfire_common rather than :bonfire
  localisation_path: "priv/localisation"

config :bonfire_common,
  otp_app: :bonfire_fail # use the current extension rather than :bonfire

## Localisation & internationalisation

config :bonfire_common, Bonfire.Common.Localise.Cldr,
  otp_app: :bonfire_fail, # use the current extension rather than :bonfire
```

In this case also removed the DB config because not used in this extension

**Comment by fishinthecalculator** [comment]
@mayel thank you. I'll fix the CI, what do you think about putting in place branch protections so that CI keeps staying green on main branch? so we could do changes such as removing the db config without breaking CI

**Comment by mayel** [comment]
@fishinthecalculator well we already have branch protections in place, but with the ability for maintainers to bypass them, which means @ivanminutillo and I currently commit and push directly to the main branches, and before changing that workflow we should discuss how it will work instead, and come up with a better process and maybe some tooling (like changes to specialised just commands like `just contrib` and/or the [script](https://github.com/bonfire-networks/bonfire-app/blob/main/git-publish.sh) it uses to prompt to commit and push changes to any locally cloned extension)...

**Comment by mayel** [comment]
Because when we work on a core feature or do some refactoring/optimisation we often have to commit changes to like 10 extensions at a time, so would be good to avoid having to manually 
- create 10 branches
- create 10 PRs
- wait for CIs
- merge 10 PRs
- switch back to the main branches on those 10 repos, etc.

**Comment by mayel** [comment]
So maybe the script could:

1. show a diff of each extension that has uncommited changes and prompt for a comment (like it does now by running `git commit --verbose --all`) and skip it otherwise
2. if a comment was provided, auto-create a branch based on it (eg `pr-refactor-feed-query-YJGA` for "Refactor feed query")
3. push the branch
4. use the `gh` command to auto-create a PR using the same comment as body 
5. switch back to main branch (though that means that the local dev instance would revert back to a pre-refactored state, which is usually not what you want...)

These are my first thoughts, as you can see I'm quite unsure about what the best approach might be. I should also point out that our tooling includes these two utilities created by a former contributor:

- [x] https://github.com/bonfire-networks/messctl which we don't really use anymore, was meant to automate the editing of our deps.hex / deps.git / deps.path files
- [x] https://github.com/bonfire-networks/jungle which I use only to `git fetch` all locally cloned extensions in parallel 

It may make sense to rewrite any useful parts of them in a stack we can more easily maintain? for messctl I think in elixir would be best, so we can integrate that automation with igniter install/upgrade scripting...

**Pull request merged**